lib/sms_sender.rb

```diff
@@ -0,0 +1,26 @@
+require 'twilio-ruby'
+
+class SmsSender
+  def initialize(sms_client = Twilio::REST::Client, time = Time)
+    @account_sid = ENV.fetch('ACCOUNT_SID')
+    @auth_token = ENV.fetch('AUTH_TOKEN')
+    @from = ENV.fetch('TWILIO_NUMBER') # Your Twilio number
+    @to = ENV.fetch('GEOFFS_NUMBER')
+    @client = sms_client.new(@account_sid, @auth_token)
+    @time = time.new
+  end
+
+  def confirm_order
+    @client.messages.create(
```

Comment:
An extension opportunity here could be to check the success of a response from twilio and potentially retry in the event of a failure